### PR TITLE
Add demo feature

### LINKED ISSUE #56: Provide a demo feature that aggregates data
Implement a simple data processing feature to demonstrate the development pipeline.

### PR DESCRIPTION
**Pull request by kretep**
solves #56

### CODE REVIEW

File: docs/notebooks/gait/gait_analysis.ipynb
Comment (vedran-kasalica) [inline docs/notebooks/gait/gait_analysis.ipynb:1]: Should we move the imports to the first cell?

`from paradigma.gait_analysis import resample_data`

File: tests/data/3.extracted_features/gait/gait_by_minute_meta.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    "device_id": "Verily Study Watch",
+    "subject_id": "X",
+    "ppp_source_protobuf": "WatchData.IMU.Week104.raw",
+    "metadata_version": "0.1",
+    "start_iso8601": "2021-06-27T16:52:20Z",
+    "end_iso8601": "2021-06-27T17:04:30Z",
+    "rows": 13,
+    "endianness": "little",
+    "data_type": "float",
+    "bits": 64,
```
Comment (vedran-kasalica) [inline tests/data/3.extracted_features/gait/gait_by_minute_meta.json:12]: Why is this not the same as the data in the method? 
` metadata_time.bits = 32`

File: src/paradigma/gait_analysis.py [hunk 2]
```diff
@@ -413,3 +413,38 @@ def quantify_arm_swing(path_to_feature_input: Union[str, Path], path_to_predicti
...
+    # Resample the data per minute (instead of second)
+    df['group'] = (df.index // 60).astype(int)
+    df_aggregated = df.groupby('group').mean()
+
+    # Prepare the metadata
+    metadata_samples.file_name = 'gait_by_minute_values.bin'
+    metadata_time.file_name = 'gait_by_minute_time.bin'
+
+    metadata_samples.channels = ['grav_accelerometer_x_mean']
+    metadata_samples.units = ['m/s^2']
```
Comment (vedran-kasalica) [inline src/paradigma/gait_analysis.py:441]:
```suggestion
    metadata_samples.units = [DataUnits.ACCELERATION]
```

File: src/paradigma/gait_analysis.py [hunk 3]
```diff
@@ -413,3 +413,38 @@ def quantify_arm_swing(path_to_feature_input: Union[str, Path], path_to_predicti
 def aggregate_weekly_arm_swing():
     pass
```
Comment (vedran-kasalica) [inline src/paradigma/gait_analysis.py:415]:
```suggestion
from paradigma.constants import DataUnits
```

File: src/paradigma/gait_analysis.py [hunk 4]
```diff
@@ -413,3 +413,38 @@ def quantify_arm_swing(path_to_feature_input: Union[str, Path], path_to_predicti
...
+
+    # Resample the data per minute (instead of second)
+    df['group'] = (df.index // 60).astype(int)
+    df_aggregated = df.groupby('group').mean()
+
+    # Prepare the metadata
+    metadata_samples.file_name = 'gait_by_minute_values.bin'
+    metadata_time.file_name = 'gait_by_minute_time.bin'
+
+    metadata_samples.channels = ['grav_accelerometer_x_mean']
```
Comment (vedran-kasalica) [inline src/paradigma/gait_analysis.py:440]: We should constants here

**Comment by kretep** [comment]
Closing PR; it was for demo purposes only

**Pull request closed without merging**